.azure/pipelines/ci.yml

```
@@ -744,3 +744,6 @@ jobs:
           artifactName: artifacts-Publish
           artifactType: Container
           parallel: true
+    - ${{ if notin(variables['Build.Reason'], 'PullRequest') }}:
```

While I agree we don't need to check for Arcade's `runAsPublic` or `disableComponentGovernance` parameters here, it's not worthwhile running the CG task in both public and internal builds.
```suggestion
    - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
```

---

This whole thing is under an `if internal` check, see line 65

---

I incorrectly thought you had updated default-build.yml instead of just the Publish job in ci.yml. Please move the task so that it affects builds for all platforms.

---

Our 2.1 builds don't really work like that - default-build.yml runs for Mac/Linux only in public PR jobs. Official jobs do a bunch of custom steps sequentially on various OSes - I put the CompGov step here since it's the job with access to all artifacts. I could also add a CompGov step to the Linux/Mac jobs that run last if we want the extra coverage.

---

Maybe the best thing would be to just add it to every one of the custom jobs in the internal pipeline

---

Ah, yes. The `default-build.yml` that is used in only a couple of unique cases 😦 Will need the job in most of the custom jobs because CG examines files like yarn.lock which aren't available in the Publish job.